Jenkinsfile

```diff
@@ -12,18 +11,7 @@ pipeline {
           agent { label 'rpmbuild' }
           steps { sh readFile(file: 'jenkins/tito-tests.sh') }
         }
-        stage('RHEL7 unit') {
-          agent { label 'subman-centos7' }
-          steps {
-            // echo "skipping for debug..."
-            sh readFile(file: 'jenkins/nose-tests.sh')
-            junit('nosetests.xml')
-            // publishCoverage('coverage.xml')
-            }
-        }
-        // TODO: figure if this is needed and implement
-        // stage('RHEL8 unit') {steps {echo 'nose'}}
-        stage('Fedora unit') {
+        stage('RHEL 8 unit') {
```

Comment:
Do we really run this on RHEL8? Or is it some Fedora?

I feel we need unit test for RHEL8 and the latest Fedora too.

What we really need is test with rpm building of RHE8 and the latest Fedora.